rflx/templates/license_header

```diff
@@ -1,8 +1,8 @@
 --
---  Generated by RecordFlux {version} on {date}
+-- Generated by RecordFlux {version} on {date}
 --
---  Copyright (C) 2018-{year} Componolit GmbH
+-- Copyright (C) 2018-{year} Componolit GmbH
```

Comment:
While we are at it: Please change that to AdaCore GmbH

Reply:
Done.